### ISSUE TITLE
Kept TIMEOUT_MAX error.

### ISSUE THREAD

**Issue by zh9118** [brentp/combined-pvalues#23]
Hi,

I was trying to run the test.py. However, it returned an error message saying the TIMEOUT_MAX error. Attached is the error image. I wonder if you can let me know what's wrong with it. Thanks.

Hua

![image](https://user-images.githubusercontent.com/13088022/74766580-380a4780-523a-11ea-913b-bf3f31b7b29c.png)

**Comment by zh9118** [comment]
BTW, I am using python 2.7.16.

**Comment by zh9118** [comment]
Hi, 

After I reinstalled Python 3, this issue disappeared. However, I got some other issues.
1) If I run comb-p pipeline, the command always kept poping out the help page.
2) If I use the python commands showed in the Github manual, the region_p.py gave me an error. Image attached.

Thanks for the help.

Hua

![image](https://user-images.githubusercontent.com/13088022/74772506-fd59dc80-5244-11ea-84fd-2d401be24b88.png)
